docs/cellxgene_census_FAQ.md

```diff
@@ -97,13 +129,13 @@ You can submit a [feature request in the github repository](https://github.com/c
 
 ## How can I contribute my data to the Census?
 
-You first need to submit your data to [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/), then it will automatically get included in the Census if it meets the [biological criteria defined in the Census schema](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/cellxgene_census_schema.md#data-included). 
+To inquire submitting your data to CZ CELLxGENE Discover you need to follow these [instructions](https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data). 
```

Comment:
```suggestion
To inquire about submitting your data to CZ CELLxGENE Discover you need to follow these [instructions](https://cellxgene.cziscience.com/docs/032__Contribute%20and%20Publish%20Data). 
```

Reply:
If I'm reading the diff correctly, this line is repeated.

Reply:
Yup I will delete the one below, thanks!